LayoutTests/imported/w3c/web-platform-tests/trusted-types/trusted-types-event-handlers-expected.txt

```diff
@@ -23,109 +116,108 @@ PASS Non-event handler div.accessKey should not be blocked.
-PASS Non-event handler div.autocorrect should not be blocked.
 PASS Non-event handler div.webkitdropzone should not be blocked.
 PASS Non-event handler div.style should not be blocked.
 PASS Non-event handler div.attributeStyleMap should not be blocked.
 PASS Non-event handler div.contentEditable should not be blocked.
-PASS Non-event handler div.enterKeyHint should not be blocked.
 PASS Non-event handler div.isContentEditable should not be blocked.
 PASS Non-event handler div.inputMode should not be blocked.
-FAIL Event handler div.onabort should be blocked. assert_throws_js: function "_ => element.setAttribute(name, "2+2")" did not throw
-FAIL Event handler div.onbeforetoggle should be blocked. assert_throws_js: function "_ => element.setAttribute(name, "2+2")" did not throw
-FAIL Event handler div.onblur should be blocked. assert_throws_js: function "_ => element.setAttribute(name, "2+2")" did not throw
-FAIL Event handler div.oncancel should be blocked. assert_throws_js: function "_ => element.setAttribute(name, "2+2")" did not throw
-FAIL Event handler div.oncanplay should be blocked. assert_throws_js: function "_ => element.setAttribute(name, "2+2")" did not throw
-FAIL Event handler div.oncanplaythrough should be blocked. assert_throws_js: function "_ => element.setAttribute(name, "2+2")" did not throw
-FAIL Event handler div.onchange should be blocked. assert_throws_js: function "_ => element.setAttribute(name, "2+2")" did not throw
-FAIL Event handler div.onclick should be blocked. assert_throws_js: function "_ => element.setAttribute(name, "2+2")" did not throw
-FAIL Event handler div.onclose should be blocked. assert_throws_js: function "_ => element.setAttribute(name, "2+2")" did not throw
-FAIL Event handler div.oncontentvisibilityautostatechange should be blocked. assert_throws_js: function "_ => element.setAttribute(name, "2+2")" did not throw
-FAIL Event handler div.oncontextmenu should be blocked. assert_throws_js: function "_ => element.setAttribute(name, "2+2")" did not throw
-FAIL Event handler div.oncopy should be blocked. assert_throws_js: function "_ => element.setAttribute(name, "2+2")" did not throw
+PASS Event handler div.onabort should be blocked.
+PASS Event handler div.onbeforetoggle should be blocked.
+PASS Event handler div.onblur should be blocked.
+PASS Event handler div.oncancel should be blocked.
+PASS Event handler div.oncanplay should be blocked.
+PASS Event handler div.oncanplaythrough should be blocked.
+PASS Event handler div.onchange should be blocked.
+PASS Event handler div.onclick should be blocked.
+PASS Event handler div.onclose should be blocked.
+PASS Event handler div.oncontentvisibilityautostatechange should be blocked.
+PASS Event handler div.oncontextmenu should be blocked.
+PASS Event handler div.oncopy should be blocked.
 FAIL Event handler div.oncuechange should be blocked. assert_throws_js: function "_ => element.setAttribute(name, "2+2")" did not throw
```

Comment:
Interesting that this one doesn't pass?

Reply:
https://searchfox.org/wubkat/source/Source/WebCore/html/HTMLElement.cpp#328

Reply:
Hmmm. Fair enough for now. Can you raise a follow up bug to look at that. If one doesn't already exist (I think there is an event handler bug already maybe just note it on that)